components/shared/DropdownButton/DropdownButton.tsx

```diff
@@ -0,0 +1,97 @@
+import { Dropdown, Image } from "../../bootstrap"
+import classNames from "classnames"
+import styled from "styled-components"
+import styles from "./dropdownButton.module.css"
+import { CarotIcon } from "./CarotIcon"
+import { Fragment } from "react"
+
+export type DropdownButtonProps = {
+  title: string
+  children: Array<string>
+  variant: string
+  styling?: stylingProps
+}
+
+export type stylingProps = {
+  width?: { desktop: string; mobile: string }
+  height?: string
+  fontSize?: string
+}
+
+const StyledDropdown = styled(Dropdown)`
+  .toggle {
+    width: ${props =>
+      props.changewidth?.desktop ? props.changewidth?.desktop : "418px"};
+    height: ${props => (props.changeheight ? props.changeheight : "40px")};
+    font-size: ${props =>
+      props.changefontsize ? props.changefontsize : "1rem"};
```

Comment:
just me being nitpicky, but could we shorten them to:
- props.changewidth?.desktop ?? "418px"
- props.changeheight ?? "40px"
- props.changefontsize ?? "1rem"

Reply:
did you try it out and see if it worked? I remember trying that and it didn't work...

Reply:
Thanks! I pushed another commit of refactoring all of the props to that.